auto-installer functionality added

msk610 opened the pull request:
Added the following method, which checks if the required components are installed then installs them automatically: 
```java
private void installRequired();
```
The method is called at `onStart()` method after `checkRequestPermissions();` it uses `SharedPreferences`, which allows developers to store key value pairs locally. The path and the keys used in the method to check whether Python and Seattle were installed are stored in global constants `DOWNLOAD_PREF_STR`, `PYTHON_KEY`, `SEATTLE_KEY`. The `ProgressDialog` class was used to give a feedback on the components being installed. Following is a screenshot of how the ProgressDialog looks:
<p align="center">
<img src ="https://cloud.githubusercontent.com/assets/10778552/23441778/6eebfa72-fdf2-11e6-94a7-efdbd0bdc8ca.png" />
</p>

msk610 committed:
modified for my environment

msk610 committed:
setup to add ui

msk610 committed:
added UI functions

msk610 committed:
docstrings

msk610 committed:
fixed imports

msk610 committed:
removed ui functions

msk610 committed:
added auto install method

msk610 committed:
updated auto-installer function to fix nullptr error

msk610 commented:
If you want to change the seattle installer to ```downloadAndInstallSeattle()``` instead of ```rawResourceInstallSeattle()``` I used raw install because the other methods sometimes crashes for me.

lukpueh commented:
@msk610 thanks for your contribution. Could you please remove the non-`SensibilityActivity.java` files from you PR?

msk610 committed:
Added tabbed layout for the different modes

Two sub layouts (fragments): Home and Manual.
- Home is the auto-installer, installs all the components automatically.
- Manual is similar to the older version of the app.

lukpueh commented:
@msk610, thanks for your awesome additions. I am going to merge your PR into yet another install-ui branch in @aaaaalbert's repo to perform some cleanup (remove the cache files, etc...).
